### PR TITLE
Updates pay it forward markdown

### PR DESCRIPTION
**Pull request by jude-lawson**

### CODE REVIEW

File: m4_sessions/1808-inning/Pay_it_forward.md [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+Andrew 
+Adam
 
 ## Name of Session: 
+Pay It Forward
 
 ## Session Description 
-
 (Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Haley Joel Osment will not be there! Join us to see and experience a main theme that runs through Turing's blood and the tech industries veins!
```
Comment (emhickmann) [inline m4_sessions/1808-inning/Pay_it_forward.md:12]: hahahahaha!  I love it.

File: m4_sessions/1808-inning/Pay_it_forward.md [hunk 2]
```diff
@@ -24,9 +29,32 @@
...
+We'll sort them on the board so they get organized into groups that are not related to the task we are teaching.
+
 ## Intro to new material
+Read the Article
+Each leader from their corner
+Each person will give brief story or connection of paying it forward in tech. 
+Justine - Give specific example in which Jude and Steve helped me and then I was able to help Jamison with connecting a FE app with a BE app.  
+Adam - leaning towards example
+Jude - Speak about paying it forward and the pivitol role it plays tech 
+Andrew - TBD
```
Comment (emhickmann) [inline m4_sessions/1808-inning/Pay_it_forward.md:43]: I really like this!  Please make sure to let Andrew know what to prep.

File: m4_sessions/1808-inning/Pay_it_forward.md [hunk 3]
```diff
@@ -24,9 +29,32 @@
...
-## Closing Discussion
+Get 4 volunteer teachers 
+Separate into groups, explain what they are going to make a first unlead attempt at while we teach the volunteers the task. 
+Then the volunteers will cascade style teach each group member, along with the previous student. 
 
+## Closing Discussion
+Leaders to 4 corners 
+Throw rubber duck around so each person reflects on what they learned or any aha moments. 
+
+Tie it together.
```
Comment (emhickmann) [inline m4_sessions/1808-inning/Pay_it_forward.md:54]: This Tie it together section is EXCELLENT!!!

File: m4_sessions/1808-inning/Pay_it_forward.md [hunk 4]
```diff
@@ -24,9 +29,32 @@
...
+Each leader from their corner
+Each person will give brief story or connection of paying it forward in tech. 
+Justine - Give specific example in which Jude and Steve helped me and then I was able to help Jamison with connecting a FE app with a BE app.  
+Adam - leaning towards example
+Jude - Speak about paying it forward and the pivitol role it plays tech 
+Andrew - TBD
+
 ## Small Group Activity
-## Closing Discussion
+Get 4 volunteer teachers
```
Comment (emhickmann) [inline m4_sessions/1808-inning/Pay_it_forward.md:46]: What do the other students do while you teach the 4?  Maybe I'm just not understanding it and y'all have the logistics in place. 

You could also take the 4 volunteers out of the room to teach them the thing...then you could work with the other students to talk about ways they have already seen Turing students/alumni pay it forward.  If you have it in a good spot, totally disregard this!

**Review by emhickmann — COMMENTED** [review]
@jude-lawson This is awesome.  Week 3 it will be important to just make sure you have logistics really solid.  I think this will be super fun!

File: m4_sessions/1808-inning/Pay_it_forward.md (hunk 4 above)
Comment (Vadlusk) [reply m4_sessions/1808-inning/Pay_it_forward.md:46]: While we teach the teacher, the other students will be attempting the challenge by themselves with no instruction to see what it's like to attempt an unknown task alone.

**Pull request merged**